### PR TITLE
Ncnews 2

### PR DESCRIPTION
**Pull request by HazemFahad**
GET /api/topics

### CODE REVIEW

**Comment by HazemFahad** [comment]
I did change husky on but for some reason removed itself - i have reinstalled now

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const db = require("../db/connection");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe.only("GET /api/topics returns all topics", () => {
+  test("endpoint returns array of topics with both description and slug", async () => {
+    const results = await request(app).get("/api/topics").expect(200);
```
Comment (xmenbteam) [inline __tests__/app.test.js:12]: Nice use of async/await!

File: app.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+app.get("/api/topics", returnTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+});
```
Comment (xmenbteam) [inline app.js:17]: Lovely.

**Review by xmenbteam — APPROVED** [review]
Well done! Good use of async/await :)

**Pull request merged**